doc/man1/openssl-verification-options.pod

```diff
@@ -240,7 +241,7 @@ chain (for example with L<openssl-s_time(1)>).
 
 =item B<-no-CAstore>
 
-Do not use the default store of trusted CA certificates.
+Do not use the default store of trusted certificates.
```

Comment:
Strictly speaking, this small change isn't quite right, is it?  All the `-CA{whatever}` options *are* about CA certificates, i.e. if you should not pass a non-CA cert (like a leaf certificate) to those options, even if you happen to trust.

I notice that you do remove CA a little here and there.  Essentially, this implies that you should be able to pass the certificate of the server you try to connect to with `s_client`.

Reply:
This change is correct. I had noticed that here the "CA" was still present while at most other such places the "CA" was already removed, for good reason: generality/completeness.

Note that a trusted certificate (serving as a trust anchor, TA) does not need to be a CA certificate.
It is an OpenSSL peculiarity that a chain by default is expected to end in a CA cert,
but in fact (see RFC 5280 etc.) any TA can do, which may also be given in the form of a directly trusted EE cert
(or by a bare public key with an associated subject name).

For instance, this does work:
```
openssl s_client -connect google.com:443 </dev/null >server-cert.pem
openssl s_client -verify_return_error -CAstore server-cert.pem -partial_chain -connect google.com:443 </dev/null
```

Reply:
So, strictly speaking, for all those CLI options and their descriptions for loading trusted certs
involving the term or name component "CA" should be changed to "TA" (trust anchor cert).

Reply:
Well, the key thing there is `partial_chain`, otherwise a chain needs to terminate a either a:

* Self-signed CA certificate (implicit trust)
* A "TRUSTED CERTIFICATE" with explicit trust EKUs.

Reply:
Yes, and the need to use `-partial_chain` for using directly trusted EE certs is what I meant with "OpenSSL peculiarity".